pkg/kv/kvserver/kvflowcontrol/rac2/testdata/range_controller/do_force_flush

```diff
@@ -32,7 +32,7 @@ t1/s3: eval reg=-3.0 MiB/+16 MiB ela=-3.0 MiB/+8.0 MiB
 
 stream_state range_id=1
 ----
-(n1,s1):1: state=replicate closed=false inflight=[1,4) send_queue=[4,4) precise_q_size=+0 B
+(n1,s1):1: state=replicate closed=false inflight=[1,4) (3.0 MiB) send_queue=[4,4) precise_q_size=+0 B
```

Comment:
For ease of review, the PR could have been split in 2 stages:

1. A mostly no-op commit that adds the in-flight volume plumbing and changes this print format to include it.
2. A commit that adds enforcement of this limit in force-flush, and corresponding test changes.

Having both together makes the diff large, and tests harder to review - lots of scrolling past the changes that are no-ops.